content/guides/examples/websites.md

```diff
@@ -119,13 +119,16 @@ you plan to use, you can find it using the command like:
 ```bash
 $ nslookup gateway.ipfs.io
 ```
-And noting the IPv4 addresses returned. You should create an A record for each address.
+1. Note the IP addresses returned.
+1. Create an A record for each IPv4 address (all results that look like 192.0.2.71).
+1. Create an AAAA record for each IPv6 address (all results that look like 2001:DB8:324A::A1).
```

Comment:
```suggestion
1. Create an AAAA record for each IPv6 address (e.g use `2602:fea2:2::1` for ipfs.io).
```

Reply:
Resolved in in https://github.com/ipfs/docs/commit/95275d753f17d5555c2b662260af1cf08061c63f